code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -364,6 +364,12 @@ GLOBAL_LIST_EMPTY(activated_medevac_stretchers)
 /obj/structure/bed/medevac_stretcher/attack_hand_alternate(mob/living/user)
 	activate_medevac_teleport(user)
 
+/obj/structure/bed/medevac_stretcher/attack_ghost(mob/dead/observer/user)
+	. = ..()
+	if(!linked_beacon || !linked_beacon.loc) //TODO: Redo medevac links
```

Comment:
This and the other medevac checks probably require a rewrite on how linking stretchers work in da future.
Outside the scope of PR therefore a TODO.